packages/metadata/tests/metadata-maps.test.ts

```diff
@@ -0,0 +1,314 @@
+import { AuthorsMap } from "../src/AuthorsMap";
+import { VariablesMap } from "../src/VariablesMap";
+import { VariableFields } from "../src/VariablesMap";
+
+let author_data = [
+  {
+    name: "John Cena",
```

Comment:
Let's not use real people

Reply:
Deleted most of the "testing data", moving functionality into system tests.